### PR TITLE
LessonOneModuleTwoHomeWorkReq

### PR DESCRIPTION
**Pull request by White4343**
Added solutions for two mandotory tasks.

### CODE REVIEW

File: SecondTaskArrayCopy/Program.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Random randomNumberInArray = new Random();
+
+            int[] A = Enumerable.Repeat(0, 20).Select(i => randomNumberInArray.Next(-888, 2000)).ToArray();
+
+            Array.Sort(A);
```
Comment (YevhenBuriak) [inline SecondTaskArrayCopy/Program.cs:11]: Since you are using linq anyway, you could use OrderBy

File: FirstTaskArrayRangeCounter/Program.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            int[] array = Enumerable.Repeat(0, 255).Select(i => randomNumberInArray.Next(-255, 255)).ToArray();
+
+            Array.Sort(array);
+
+            Console.WriteLine($"Newly created array:");
+            Console.WriteLine("[{0}]", string.Join(", ", array));
+            
+            byte j = 0;
+
+            foreach (var i in array.Where(i => i >= -100 & i <= 100))
```
Comment (YevhenBuriak) [inline FirstTaskArrayRangeCounter/Program.cs:18]: Why & and not &&?

**Review by YevhenBuriak — APPROVED** [review]
Left some comments, you may take a look and merge.

File: FirstTaskArrayRangeCounter/Program.cs (hunk 2 above)
Comment (White4343) [reply FirstTaskArrayRangeCounter/Program.cs:18]: IDE throws an error with using &&.

**Pull request merged**